components/Badge/badge.html

```diff
@@ -20,16 +22,32 @@
     </nav>
     <main class="doc-container">
         <aside class="side-nav">
-            <h3>Get Started</h3>
-            <ul>
+            <div class="side-nav-header">
+                <h3>Get Started</h3>
+                <span id="get-started-expand-more" class="material-icons-outlined display-block" onclick="getStartedExpandMore()">
+                    expand_more
+                </span>
+                <span id="get-started-expand-less" class="material-icons-outlined display-none" onclick="getStartedExpandLess()">
+                    expand_less
```

Comment:
I think it was mentioned in level zero that using onclick like this is a bad idea and we should use addeventlistener instead , no need to refactor as u don't have time , but just wanted to let u know for future